src/main/kotlin/ink/abb/pogo/scraper/Bot.kt

```diff
@@ -98,8 +98,11 @@ class Bot(val api: PokemonGo, val settings: Settings) {
         runningLatch = CountDownLatch(1)
         phaser = Phaser(1)
 
-        runLoop(TimeUnit.SECONDS.toMillis(60), "ProfileLoop") {
+        runLoop(TimeUnit.SECONDS.toMillis(settings.profileUpdateTimer), "ProfileLoop") {
             task(profile)
+        }
+		
+	runLoop(TimeUnit.SECONDS.toMillis(60), "HatchEggsLoop") {
             task(hatchEggs)
```

Comment:
not sure if hatchEggs relies on the profile update (it does need a fresh inventory). If that's the case, please remove this runLoop and put the eggs behind the task(profile) again